src/igvfd/schemas/tabular_file.json

```diff
@@ -234,7 +237,9 @@
                 "element to gene interactions": "A file summarizing the results of an E2G based prediction to identify the linkage between genomic elements and genes.",
                 "elements reference": "References for the genomic elements studied, including details like genomic coordinates or identifiers.",
                 "fold change over control": "The ratio of expression levels or other measurements in an experimental condition relative to a control.",
+                "fold change of post selection and pre selection": "The log2 fold change of the variant allele ratio (variant reads divided by reference reads) between post-selection and pre-selection for each variant.",
```

Comment:
do you want to add a hyphen there? fold change of post**-**selection and pre**-**selection